ui/article-renderer/src/main/res/layout/fragment_articles.xml

```diff
@@ -1,11 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.v7.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.v4.widget.SwipeRefreshLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/article_swipe_container"
+    android:layout_width="match_parent"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/articles_recycler_view"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/white"
-    android:theme="@style/ThemeOverlay.Widget.GodTools.RecyclerView.Articles"
-    app:layoutManager="android.support.v7.widget.LinearLayoutManager"
-    tools:listitem="@layout/list_item_article" />
+    android:layout_height="match_parent">
+
+    <android.support.v7.widget.RecyclerView
+        android:id="@+id/articles_recycler_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/white"
+        android:theme="@style/ThemeOverlay.Widget.GodTools.RecyclerView.Articles"
+        app:layoutManager="android.support.v7.widget.LinearLayoutManager"
+        tools:listitem="@layout/list_item_article" />
+</android.support.v4.widget.SwipeRefreshLayout>
```

Comment:
add a trailing line return. it's a good practice/habit to get into because without it various command line utilities (including `git diff`) behave weirdly. See: https://unix.stackexchange.com/a/18789